defaults/main.yml

```diff
@@ -552,7 +548,7 @@ maas_pip_packages:
   - python-keystoneclient
   - python-magnumclient
   - python-neutronclient
-  - python-novaclient<7.0.0
+  - python-novaclient
```

Comment:
if we care about using the same version as whatever branch we are on we should tie the pip install to upper-constraints.txt